diff -dur crossfire-1.5.0.20030416.orig/acinclude.m4 crossfire-1.5.0.20030416/acinclude.m4
--- crossfire-1.5.0.20030416.orig/acinclude.m4	2003-04-05 07:39:46.000000000 +0200
+++ crossfire-1.5.0.20030416/acinclude.m4	2004-04-16 18:53:42.867327547 +0200
@@ -7,7 +7,7 @@
 	dir=""
 	AC_CHECK_HEADERS([Python.h],[cf_have_python_h=yes])
 	if test "x$cf_have_python_h" = "x"  ; then
-		for dir in /usr{,/local}/include/python{,2.2,2.1,2.0} ; do
+		for dir in /usr{,/local}/include/python{,2.3,2.2,2.1,2.0} ; do
 			AC_CHECK_HEADERS(["$dir/Python.h"],[cf_have_python_h=yes])
 			if test "x$cf_have_python_h" != "x" ; then
 				PY_INCLUDES="-I$dir"
@@ -21,7 +21,7 @@
 	if test "x$cf_have_python_h" = "xyes" ; then
 		PYTHON_LIB=""
 
-                for lib in python{,2.2,2.1,2.0} ; do
+                for lib in python{,2.3,2.2,2.1,2.0} ; do
                         AC_CHECK_LIB($lib, PyArg_ParseTuple,[PYTHON_LIB="-l$lib"])
                         if test "x$PYTHON_LIB" != "x" ; then
                                 break
