--- crossfire-1.3.0/random_maps/Makefile.in.wiget	Tue Jul  2 01:18:12 2002
+++ crossfire-1.3.0/random_maps/Makefile.in	Mon Jul  8 21:19:04 2002
@@ -125,10 +125,10 @@
 	@echo $(SRCS) > .patchlist
 
 install:: random_map
-	@if [ ! -d ${bindir} ]; then \
-		$(MKDIR) -p ${bindir}; \
+	@if [ ! -d $(DESTDIR)${bindir} ]; then \
+		$(MKDIR) -p $(DESTDIR)${bindir}; \
 	fi
-	$(INSTALL) random_map ${bindir}/random_map
+	$(INSTALL) random_map $(DESTDIR)${bindir}/random_map
 
 
 
--- crossfire-1.3.0/server/Makefile.in.wiget	Tue Jul  2 01:18:12 2002
+++ crossfire-1.3.0/server/Makefile.in	Mon Jul  8 21:19:04 2002
@@ -139,7 +139,7 @@
 	@echo $(SRCS) > .patchlist
 
 install:: crossfire
-	@if [ ! -d ${bindir} ]; then \
-		$(MKDIR) -p ${bindir}; \
+	@if [ ! -d $(DESTDIR)${bindir} ]; then \
+		$(MKDIR) -p $(DESTDIR)${bindir}; \
 	fi
-	$(INSTALL) crossfire ${bindir}/crossfire
+	$(INSTALL) crossfire $(DESTDIR)${bindir}/crossfire
--- crossfire-1.3.0/lib/Makefile.in.wiget	Tue Jul  2 01:18:12 2002
+++ crossfire-1.3.0/lib/Makefile.in	Mon Jul  8 21:25:19 2002
@@ -131,58 +131,58 @@
 	make collect
 
 install:: archetypes bmaps.paths crossfire.0 faces animations collect.pl util.pl
-	@if [ ! -d $(datadir) ]; then \
-		$(MKDIR) -p $(datadir); \
+	@if [ ! -d $(DESTDIR)$(datadir) ]; then \
+		$(MKDIR) -p $(DESTDIR)$(datadir); \
 	fi
-	@if [ ! -d $(localdir) ]; then \
-		$(MKDIR) -p $(localdir); \
+	@if [ ! -d $(DESTDIR)$(localdir) ]; then \
+		$(MKDIR) -p $(DESTDIR)$(localdir); \
 	fi
 
-	@if [ ! -d $(datadir)/help ]; then \
-		echo "Creating directory $(datadir)/help"; \
-		$(MKDIR) -p $(datadir)/help; \
+	@if [ ! -d $(DESTDIR)$(datadir)/help ]; then \
+		echo "Creating directory $(DESTDIR)$(datadir)/help"; \
+		$(MKDIR) -p $(DESTDIR)$(datadir)/help; \
 	fi;
 	@for i in $(FILES_HELP); do \
 	    echo "Installing $$i"; \
-	    $(INSTALL) $(INSTALLFLAGS) $$i $(datadir)/help; \
+	    $(INSTALL) $(INSTALLFLAGS) $$i $(DESTDIR)$(datadir)/help; \
 	done
-	@if [ ! -d $(datadir)/wizhelp ]; then \
-		echo "Creating directory $(datadir)/wizhelp"; \
-		$(MKDIR) -p $(datadir)/wizhelp; \
+	@if [ ! -d $(DESTDIR)$(datadir)/wizhelp ]; then \
+		echo "Creating directory $(DESTDIR)$(datadir)/wizhelp"; \
+		$(MKDIR) -p $(DESTDIR)$(datadir)/wizhelp; \
 	fi;
 	@for i in $(FILES_WIZHELP); do \
 	    echo "Installing $$i"; \
-	    $(INSTALL) $(INSTALLFLAGS) $$i $(datadir)/wizhelp; \
+	    $(INSTALL) $(INSTALLFLAGS) $$i $(DESTDIR)$(datadir)/wizhelp; \
 	done
 	@for i in $(INSTALLFILES); do \
 	    echo "Installing $$i"; \
-	    $(INSTALL) $(INSTALLFLAGS)  $$i $(datadir); \
+	    $(INSTALL) $(INSTALLFLAGS)  $$i $(DESTDIR)$(datadir); \
 	done
 
 	@for i in $(BININSTALL); do \
 		echo "Installing $$i"; \
-		$(INSTALL) $(INSTALLFLAGS)  $$i $(bindir); \
+		$(INSTALL) $(INSTALLFLAGS)  $$i $(DESTDIR)$(bindir); \
 	done
 
-	@if [ ! -f $(datadir)/settings ]; then \
-		$(INSTALL) $(INSTALLFLAGS) settings $(datadir); \
+	@if [ ! -f $(DESTDIR)$(datadir)/settings ]; then \
+		$(INSTALL) $(INSTALLFLAGS) settings $(DESTDIR)$(datadir); \
 	fi
 
 	@for i in $(INSTALLNEWFILES); do \
-	    if [ ! -f $(datadir)/$$i ]; then \
+	    if [ ! -f $(DESTDIR)$(datadir)/$$i ]; then \
 		    echo "Installing $$i"; \
-		    $(INSTALL) $(INSTALLFLAGS)  $$i $(datadir); \
+		    $(INSTALL) $(INSTALLFLAGS)  $$i $(DESTDIR)$(datadir); \
 	    fi \
 	done
 
-	touch $(localdir)/highscore
-	touch $(localdir)/bookarch
-	touch $(localdir)/temp.maps
+	touch $(DESTDIR)$(localdir)/highscore
+	touch $(DESTDIR)$(localdir)/bookarch
+	touch $(DESTDIR)$(localdir)/temp.maps
 
 	@for i in players unique-items; do \
-	    if [ ! -d $(localdir)/$$i ]; then \
-		echo "Creating directory $(localdir)/$$i"; \
-		$(MKDIR) -p $(localdir)/$$i; \
+	    if [ ! -d $(DESTDIR)$(localdir)/$$i ]; then \
+		echo "Creating directory $(DESTDIR)$(localdir)/$$i"; \
+		$(MKDIR) -p $(DESTDIR)$(localdir)/$$i; \
 	    fi; \
 	done
 
--- crossfire-1.3.0/utils/Makefile.in.wiget	Tue Jul  2 01:18:21 2002
+++ crossfire-1.3.0/utils/Makefile.in	Mon Jul  8 21:27:34 2002
@@ -20,22 +20,22 @@
 	$(DEPEND) $(DEPENDFLAGS) -- $(ALLDEFINES) $(DEPEND_DEFINES) -- $(SRCS)
 
 install:
-	@if [ ! -d $(bindir) ]; then \
-		$(MKDIR) -p $(bindir); \
+	@if [ ! -d $(DESTDIR)$(bindir) ]; then \
+		$(MKDIR) -p $(DESTDIR)$(bindir); \
 	fi
-	rm -f $(bindir)/flushlocks
-	sed s%xxxx%$(libdir)%g <flushlocks > $(bindir)/flushlocks
-	chmod 755 $(bindir)/flushlocks
-	rm -f $(bindir)/crossloop.pl
-	sed s%BINDIR%$(bindir)%g <crossloop.pl >$(bindir)/crossloop.pl
-	chmod 755 $(bindir)/crossloop.pl
-	rm -f $(bindir)/crossloop
-	sed s%BINDIR%$(bindir)%g <crossloop >$(bindir)/crossloop
-	chmod 755 $(bindir)/crossloop
-	$(INSTALL) $(INSTALLFLAGS) mktable.script $(bindir)/mktable
-	$(INSTALL) $(INSTALLFLAGS) add_throw.perl $(bindir)/add_throw
-	@if [ ! -f $(bindir)/crossloop.web ]; then \
-		$(INSTALL) $(INSTALLFLAGS) crossloop.web $(bindir)/crossloop.web; \
+	rm -f $(DESTDIR)$(bindir)/flushlocks
+	sed s%xxxx%$(libdir)%g <flushlocks > $(DESTDIR)$(bindir)/flushlocks
+	chmod 755 $(DESTDIR)$(bindir)/flushlocks
+	rm -f $(DESTDIR)$(bindir)/crossloop.pl
+	sed s%BINDIR%$(bindir)%g <crossloop.pl >$(DESTDIR)$(bindir)/crossloop.pl
+	chmod 755 $(DESTDIR)$(bindir)/crossloop.pl
+	rm -f $(DESTDIR)$(bindir)/crossloop
+	sed s%BINDIR%$(bindir)%g <crossloop >$(DESTDIR)$(bindir)/crossloop
+	chmod 755 $(DESTDIR)$(bindir)/crossloop
+	$(INSTALL) $(INSTALLFLAGS) mktable.script $(DESTDIR)$(bindir)/mktable
+	$(INSTALL) $(INSTALLFLAGS) add_throw.perl $(DESTDIR)$(bindir)/add_throw
+	@if [ ! -f $(DESTDIR)$(bindir)/crossloop.web ]; then \
+		$(INSTALL) $(INSTALLFLAGS) crossloop.web $(DESTDIR)$(bindir)/crossloop.web; \
 	fi
 
 depend::
--- crossfire-1.3.0/doc/Makefile.in.wiget	Tue Jul  2 01:18:21 2002
+++ crossfire-1.3.0/doc/Makefile.in	Mon Jul  8 21:19:04 2002
@@ -106,8 +106,8 @@
 	$(RM) -rf $(DTOP)
 
 install::
-	@if [ ! -d $(MAN1) ]; then \
-		$(MKDIR) -p $(MAN1); \
+	@if [ ! -d $(DESTDIR)$(MAN1) ]; then \
+		$(MKDIR) -p $(DESTDIR)$(MAN1); \
 	fi
-	$(INSTALL) $(srcdir)/crossedit.man $(MAN1)/crossedit.$(MANSUFFIX)
-	$(INSTALL) $(srcdir)/crossfire.man $(MAN1)/crossfire.$(MANSUFFIX)
+	$(INSTALL) $(srcdir)/crossedit.man $(DESTDIR)$(MAN1)/crossedit.$(MANSUFFIX)
+	$(INSTALL) $(srcdir)/crossfire.man $(DESTDIR)$(MAN1)/crossfire.$(MANSUFFIX)
--- crossfire-1.3.0/crossedit/Makefile.in.wiget	Tue Jul  2 01:18:22 2002
+++ crossfire-1.3.0/crossedit/Makefile.in	Mon Jul  8 21:19:04 2002
@@ -150,7 +150,7 @@
 
 
 install:: crossedit
-	@if [ ! -d ${bindir} ]; then \
-		$(MKDIR) -p ${bindir}; \
+	@if [ ! -d $(DESTDIR)${bindir} ]; then \
+		$(MKDIR) -p $(DESTDIR)${bindir}; \
 	fi
-	$(INSTALL) crossedit ${bindir}/crossedit
+	$(INSTALL) crossedit $(DESTDIR)${bindir}/crossedit
--- crossfire-1.3.0/Makefile.in.wiget	Tue Jul  2 01:18:12 2002
+++ crossfire-1.3.0/Makefile.in	Mon Jul  8 21:19:04 2002
@@ -124,7 +124,7 @@
 
 install::
 	@if [ ! -d ${prefix} ]; then \
-		$(MKDIR) -p ${prefix}; \
+		$(MKDIR) -p $(DESTDIR)${prefix}; \
 	fi
 	@for dir in $(SUBDIRS); \
 	do \
--- crossfire-1.3.0/plugin/Makefile.in.destdir	Mon Jul  8 21:43:03 2002
+++ crossfire-1.3.0/plugin/Makefile.in	Mon Jul  8 21:44:19 2002
@@ -65,12 +65,12 @@
 # as such, if plugin target is set, then we install, otherwise 
 # we don't do anything.
 install:
-	@if [ ! -d $(CROSSFIRE_PATH)/plugins ]; then \
-	    echo "Creating directory $(CROSSFIRE_PATH)/plugins"; \
-	    $(MKDIR) -p $(CROSSFIRE_PATH)/plugins; \
+	@if [ ! -d $(DESTDIR)$(CROSSFIRE_PATH)/plugins ]; then \
+	    echo "Creating directory $(DESTDIR)$(CROSSFIRE_PATH)/plugins"; \
+	    $(MKDIR) -p $(DESTDIR)$(CROSSFIRE_PATH)/plugins; \
 	fi
 	@if [ "X$(PLUGIN_TARGET)" != "X" ]; then \
-		$(INSTALL) plugin_python.so.0.1 $(CROSSFIRE_PATH)/plugins; \
+		$(INSTALL) plugin_python.so.0.1 $(DESTDIR)$(CROSSFIRE_PATH)/plugins; \
 	fi
 
 plugin_python.so.0.1: $(OBJS) ../common/libcross.a
