--- crossfire-1.10.0/server/daemon.c.wig2	2006-05-13 08:32:14.000000000 +0200
+++ crossfire-1.10.0/server/daemon.c	2007-02-04 14:22:56.000000000 +0100
@@ -104,8 +104,8 @@ FILE *become_daemon (const char *filenam
      * Set up the standard file descriptors.
      */
     (void) open ("/dev/null", O_RDONLY);	/* root inode already in core */
-    (void) dup2 (0, 1);
-    (void) dup2 (0, 2);
+    (void) dup2 (fileno(logfile), 1);
+    (void) dup2 (fileno(logfile), 2);
 
     if ((i = open ("/dev/tty", O_RDWR)) >= 0) {	/* did open succeed? */
 #if (defined(SYSV) || defined(hpux)) && defined(TIOCTTY)
