diff -durN crossfire-0.98.0.orig/include/config.h crossfire-0.98.0/include/config.h
--- crossfire-0.98.0.orig/include/config.h	Sun Apr  8 07:49:34 2001
+++ crossfire-0.98.0/include/config.h	Tue Apr 17 09:45:40 2001
@@ -680,7 +680,7 @@
 #ifdef WIN32 /* change define path */
 #define LOGFILE "tmp\\cross.log"
 #else 
-#define LOGFILE "/tmp/cross.log"
+#define LOGFILE "/var/log/crossfire"
 #endif
 #endif
 
@@ -830,7 +830,7 @@
 #ifdef WIN32 /* change define path tmp */
 #define TMPDIR "tmp"
 #else
-#define TMPDIR "/tmp"
+#define TMPDIR "/var/lib/crossfire/tmp"
 #endif
 
 
diff -durN crossfire-0.98.0.orig/server/main.c crossfire-0.98.0/server/main.c
--- crossfire-0.98.0.orig/server/main.c	Sun Apr  8 07:49:34 2001
+++ crossfire-0.98.0/server/main.c	Tue Apr 17 09:44:01 2001
@@ -29,6 +29,7 @@
 #include <version.h>
 #include <global.h>
 #include <object.h>
+#include <config.h>
 
 #ifdef HAVE_DES_H
 #include <des.h>
