diff -durN crossfire-1.1.0.orig/Makefile.in crossfire-1.1.0/Makefile.in
--- crossfire-1.1.0.orig/Makefile.in	Wed Mar 20 07:36:53 2002
+++ crossfire-1.1.0/Makefile.in	Fri Mar 22 13:14:37 2002
@@ -124,7 +124,7 @@
 
 install::
 	@if [ ! -d ${prefix} ]; then \
-		$(MKDIR) -p ${prefix}; \
+		$(MKDIR) -p $(DESTDIR)${prefix}; \
 	fi
 	@for dir in $(SUBDIRS); \
 	do \
diff -durN crossfire-1.1.0.orig/crossedit/Makefile.in crossfire-1.1.0/crossedit/Makefile.in
--- crossfire-1.1.0.orig/crossedit/Makefile.in	Wed Mar 20 07:36:54 2002
+++ crossfire-1.1.0/crossedit/Makefile.in	Fri Mar 22 13:14:37 2002
@@ -150,7 +150,7 @@
 
 
 install:: crossedit
-	@if [ ! -d ${bindir} ]; then \
-		$(MKDIR) -p ${bindir}; \
+	@if [ ! -d $(DESTDIR)${bindir} ]; then \
+		$(MKDIR) -p $(DESTDIR)${bindir}; \
 	fi
-	$(INSTALL) crossedit ${bindir}/crossedit
+	$(INSTALL) crossedit $(DESTDIR)${bindir}/crossedit
diff -durN crossfire-1.1.0.orig/doc/Makefile.in crossfire-1.1.0/doc/Makefile.in
--- crossfire-1.1.0.orig/doc/Makefile.in	Wed Mar 20 07:36:54 2002
+++ crossfire-1.1.0/doc/Makefile.in	Fri Mar 22 13:14:37 2002
@@ -106,8 +106,8 @@
 	$(RM) -rf $(DTOP)
 
 install::
-	@if [ ! -d $(MAN1) ]; then \
-		$(MKDIR) -p $(MAN1); \
+	@if [ ! -d $(DESTDIR)$(MAN1) ]; then \
+		$(MKDIR) -p $(DESTDIR)$(MAN1); \
 	fi
-	$(INSTALL) $(srcdir)/crossedit.man $(MAN1)/crossedit.$(MANSUFFIX)
-	$(INSTALL) $(srcdir)/crossfire.man $(MAN1)/crossfire.$(MANSUFFIX)
+	$(INSTALL) $(srcdir)/crossedit.man $(DESTDIR)$(MAN1)/crossedit.$(MANSUFFIX)
+	$(INSTALL) $(srcdir)/crossfire.man $(DESTDIR)$(MAN1)/crossfire.$(MANSUFFIX)
diff -durN crossfire-1.1.0.orig/lib/Makefile.in crossfire-1.1.0/lib/Makefile.in
--- crossfire-1.1.0.orig/lib/Makefile.in	Wed Mar 20 07:36:54 2002
+++ crossfire-1.1.0/lib/Makefile.in	Fri Mar 22 13:16:31 2002
@@ -126,51 +126,51 @@
 	make collect
 
 install:: archetypes bmaps.paths crossfire.0 animations collect.pl util.pl
-	@if [ ! -d $(datadir) ]; then \
-		$(MKDIR) -p $(datadir); \
+	@if [ ! -d $(DESTDIR)$(datadir) ]; then \
+		$(MKDIR) -p $(DESTDIR)$(datadir); \
 	fi
-	@if [ ! -d $(localdir) ]; then \
-		$(MKDIR) -p $(localdir); \
+	@if [ ! -d $(DESTDIR)$(localdir) ]; then \
+		$(MKDIR) -p $(DESTDIR)$(localdir); \
 	fi
 
-	if [ ! -d $(datadir)/help ]; then \
-		echo "Creating directory $(datadir)/help"; \
-		$(MKDIR) -p $(datadir)/help; \
+	if [ ! -d $(DESTDIR)$(datadir)/help ]; then \
+		echo "Creating directory $(DESTDIR)$(datadir)/help"; \
+		$(MKDIR) -p $(DESTDIR)$(datadir)/help; \
 	fi;
 	@for i in $(FILES_HELP); do \
 	    echo "Installing $$i"; \
-	    $(INSTALL) $(INSTALLFLAGS) $$i $(datadir)/help; \
+	    $(INSTALL) $(INSTALLFLAGS) $$i $(DESTDIR)$(datadir)/help; \
 	done
-	if [ ! -d $(datadir)/wizhelp ]; then \
-		echo "Creating directory $(datadir)/wizhelp"; \
-		$(MKDIR) -p $(datadir)/wizhelp; \
+	if [ ! -d $(DESTDIR)$(datadir)/wizhelp ]; then \
+		echo "Creating directory $(DESTDIR)$(datadir)/wizhelp"; \
+		$(MKDIR) -p $(DESTDIR)$(datadir)/wizhelp; \
 	fi;
 	@for i in $(FILES_WIZHELP); do \
 	    echo "Installing $$i"; \
-	    $(INSTALL) $(INSTALLFLAGS) $$i $(datadir)/wizhelp; \
+	    $(INSTALL) $(INSTALLFLAGS) $$i $(DESTDIR)$(datadir)/wizhelp; \
 	done
 	@for i in $(INSTALLFILES); do \
 	    echo "Installing $$i"; \
-	    $(INSTALL) $(INSTALLFLAGS)  $$i $(datadir); \
+	    $(INSTALL) $(INSTALLFLAGS)  $$i $(DESTDIR)$(datadir); \
 	done
 
 	@for i in $(BININSTALL); do \
 		echo "Installing $$i"; \
-		$(INSTALL) $(INSTALLFLAGS)  $$i $(bindir); \
+		$(INSTALL) $(INSTALLFLAGS)  $$i $(DESTDIR)$(bindir); \
 	done
 
 	@if [ ! -f $(datadir)/settings ]; then \
-		$(INSTALL) $(INSTALLFLAGS) settings $(datadir); \
+		$(INSTALL) $(INSTALLFLAGS) settings $(DESTDIR)$(datadir); \
 	fi
 
-	touch $(localdir)/highscore
-	touch $(localdir)/bookarch
-	touch $(localdir)/temp.maps
+	touch $(DESTDIR)$(localdir)/highscore
+	touch $(DESTDIR)$(localdir)/bookarch
+	touch $(DESTDIR)$(localdir)/temp.maps
 
 	@for i in players unique-items; do \
 	    if [ ! -d $(localdir)/$$i ]; then \
-		echo "Creating directory $(localdir)/$$i"; \
-		$(MKDIR) -p $(localdir)/$$i; \
+		echo "Creating directory $(DESTDIR)$(localdir)/$$i"; \
+		$(MKDIR) -p $(DESTDIR)$(localdir)/$$i; \
 	    fi; \
 	done
 
diff -durN crossfire-1.1.0.orig/random_maps/Makefile.in crossfire-1.1.0/random_maps/Makefile.in
--- crossfire-1.1.0.orig/random_maps/Makefile.in	Wed Mar 20 07:36:53 2002
+++ crossfire-1.1.0/random_maps/Makefile.in	Fri Mar 22 13:14:37 2002
@@ -123,10 +123,10 @@
 	@echo $(SRCS) > .patchlist
 
 install:: random_map
-	@if [ ! -d ${bindir} ]; then \
-		$(MKDIR) -p ${bindir}; \
+	@if [ ! -d $(DESTDIR)${bindir} ]; then \
+		$(MKDIR) -p $(DESTDIR)${bindir}; \
 	fi
-	$(INSTALL) random_map ${bindir}/random_map
+	$(INSTALL) random_map $(DESTDIR)${bindir}/random_map
 
 
 
diff -durN crossfire-1.1.0.orig/server/Makefile.in crossfire-1.1.0/server/Makefile.in
--- crossfire-1.1.0.orig/server/Makefile.in	Wed Mar 20 07:36:53 2002
+++ crossfire-1.1.0/server/Makefile.in	Fri Mar 22 13:14:37 2002
@@ -139,7 +139,7 @@
 	@echo $(SRCS) > .patchlist
 
 install:: crossfire
-	@if [ ! -d ${bindir} ]; then \
-		$(MKDIR) -p ${bindir}; \
+	@if [ ! -d $(DESTDIR)${bindir} ]; then \
+		$(MKDIR) -p $(DESTDIR)${bindir}; \
 	fi
-	$(INSTALL) crossfire ${bindir}/crossfire
+	$(INSTALL) crossfire $(DESTDIR)${bindir}/crossfire
diff -durN crossfire-1.1.0.orig/utils/Makefile.in crossfire-1.1.0/utils/Makefile.in
--- crossfire-1.1.0.orig/utils/Makefile.in	Wed Mar 20 07:36:54 2002
+++ crossfire-1.1.0/utils/Makefile.in	Fri Mar 22 13:17:35 2002
@@ -20,20 +20,20 @@
 	$(DEPEND) $(DEPENDFLAGS) -- $(ALLDEFINES) $(DEPEND_DEFINES) -- $(SRCS)
 
 install:
-	@if [ ! -d $(bindir) ]; then \
-		$(MKDIR) -p $(bindir); \
+	@if [ ! -d $(DESTDIR)$(bindir) ]; then \
+		$(MKDIR) -p $(DESTDIR)$(bindir); \
 	fi
-	rm -f $(bindir)/flushlocks
-	sed s%xxxx%$(libdir)%g <flushlocks > $(bindir)/flushlocks
-	chmod 755 $(bindir)/flushlocks
-	rm -f $(bindir)/crossloop.pl
-	sed s%BINDIR%$(bindir)%g <crossloop.pl >$(bindir)/crossloop.pl
-	chmod 755 $(bindir)/crossloop.pl
-	rm -f $(bindir)/crossloop
-	sed s%BINDIR%$(bindir)%g <crossloop >$(bindir)/crossloop
-	chmod 755 $(bindir)/crossloop
-	$(INSTALL) $(INSTALLFLAGS) mktable.script $(bindir)/mktable
-	$(INSTALL) $(INSTALLFLAGS) add_throw.perl $(bindir)/add_throw
+	rm -f $(DESTDIR)$(bindir)/flushlocks
+	sed s%xxxx%$(libdir)%g <flushlocks > $(DESTDIR)$(bindir)/flushlocks
+	chmod 755 $(DESTDIR)$(bindir)/flushlocks
+	rm -f $(DESTDIR)$(bindir)/crossloop.pl
+	sed s%BINDIR%$(bindir)%g <crossloop.pl >$(DESTDIR)$(bindir)/crossloop.pl
+	chmod 755 $(DESTDIR)$(bindir)/crossloop.pl
+	rm -f $(DESTDIR)$(bindir)/crossloop
+	sed s%BINDIR%$(bindir)%g <crossloop >$(DESTDIR)$(bindir)/crossloop
+	chmod 755 $(DESTDIR)$(bindir)/crossloop
+	$(INSTALL) $(INSTALLFLAGS) mktable.script $(DESTDIR)$(bindir)/mktable
+	$(INSTALL) $(INSTALLFLAGS) add_throw.perl $(DESTDIR)$(bindir)/add_throw
 
 depend::
 
