diff -durN crossfire-1.1.0.orig/include/config.h crossfire-1.1.0/include/config.h
--- crossfire-1.1.0.orig/include/config.h	Wed Mar 20 07:36:53 2002
+++ crossfire-1.1.0/include/config.h	Fri Mar 22 13:23:04 2002
@@ -654,7 +654,7 @@
 #ifdef WIN32 /* change define path */
 #define LOGFILE "tmp\\cross.log"
 #else 
-#define LOGFILE "/tmp/cross.log"
+#define LOGFILE "/var/log/crossfire"
 #endif
 #endif
 
@@ -821,7 +821,7 @@
 #ifdef WIN32 /* change define path tmp */
 #define TMPDIR "tmp"
 #else
-#define TMPDIR "/tmp"
+#define TMPDIR "/var/lib/crossfire/tmp"
 #endif
 
 
diff -durN crossfire-1.1.0.orig/server/ban.c crossfire-1.1.0/server/ban.c
--- crossfire-1.1.0.orig/server/ban.c	Wed Mar 20 07:36:53 2002
+++ crossfire-1.1.0/server/ban.c	Fri Mar 22 13:23:04 2002
@@ -34,7 +34,7 @@
   int           num1;
   int   Hits=0;                 /* Hits==2 means we're banned */
 
-  sprintf (buf, "%s/%s", settings.datadir, BANFILE);
+  sprintf (buf, "%s/%s", "/etc/crossfire", BANFILE);
   if ((bannedfile = fopen(buf, "r")) == NULL) {
     LOG (llevDebug, "Could not find file Banned file.\n");
     return(0);
diff -durN crossfire-1.1.0.orig/server/c_wiz.c crossfire-1.1.0/server/c_wiz.c
--- crossfire-1.1.0.orig/server/c_wiz.c	Wed Mar 20 07:36:53 2002
+++ crossfire-1.1.0/server/c_wiz.c	Fri Mar 22 13:23:04 2002
@@ -889,7 +889,7 @@
   *pl_name=op->name?op->name:"*";
 #endif
 
-  sprintf (buf, "%s/%s", settings.datadir, DMFILE);
+  sprintf (buf, "%s/%s", "/etc/crossfire", DMFILE);
   if ((dmfile = fopen(buf, "r")) == NULL) {
     LOG (llevDebug, "Could not find DM file.\n");
     return(0);
diff -durN crossfire-1.1.0.orig/server/init.c crossfire-1.1.0/server/init.c
--- crossfire-1.1.0.orig/server/init.c	Wed Mar 20 07:36:53 2002
+++ crossfire-1.1.0/server/init.c	Fri Mar 22 13:23:04 2002
@@ -225,7 +225,7 @@
     int	has_val,comp;
     FILE    *fp;
 
-    sprintf(buf,"%s/settings",settings.datadir);
+    sprintf(buf,"%s/settings","/etc/crossfire");
     /* We don't require a settings file at current time, but down the road,
      * there will probably be so many values that not having a settings file
      * will not be a good thing.
@@ -504,7 +504,7 @@
   int comp;
 
 #ifdef SHUTDOWN_FILE
-  sprintf(buf,"%s/%s",settings.datadir,SHUTDOWN_FILE);
+  sprintf(buf,"%s/%s","/etc/crossfire",SHUTDOWN_FILE);
   if ((fp = open_and_uncompress(buf, 0, &comp)) != NULL) {
     while (fgets(buf, MAX_BUF-1, fp) != NULL)
       printf("%s", buf);
@@ -558,10 +558,10 @@
   printf("Datadir:\t\t%s\n",settings.datadir);
   printf("Localdir:\t\t%s\n",settings.localdir);
 #ifdef PERM_FILE
-  printf("Perm file:\t<LIB>/%s\n",PERM_FILE);
+  printf("Perm file:\t/etc/crossfire/%s\n",PERM_FILE);
 #endif
 #ifdef SHUTDOWN_FILE
-  printf("Shutdown file:\t<LIB>/%s\n",SHUTDOWN_FILE);
+  printf("Shutdown file:\t/etc/crossfire/%s\n",SHUTDOWN_FILE);
 #endif
   printf("Save player:\t<true>\n");
   printf("Save mode:\t%4.4o\n",SAVE_MODE);
diff -durN crossfire-1.1.0.orig/server/main.c crossfire-1.1.0/server/main.c
--- crossfire-1.1.0.orig/server/main.c	Wed Mar 20 07:36:53 2002
+++ crossfire-1.1.0/server/main.c	Fri Mar 22 13:23:43 2002
@@ -30,6 +30,7 @@
 #include <global.h>
 #include <object.h>
 #include <tod.h>
+#include <config.h>
 
 #ifdef HAVE_DES_H
 #include <des.h>
@@ -1043,7 +1044,7 @@
     clock = time (NULL);
     tm = (struct tm *) localtime (&clock);
 
-    sprintf (buf, "%s/%s", settings.datadir, PERM_FILE);
+    sprintf (buf, "%s/%s", "/etc/crossfire", PERM_FILE);
     if ((fp = open_and_uncompress(buf, 0, &comp)) == NULL)
 	return 0;
 
diff -durN crossfire-1.1.0.orig/server/player.c crossfire-1.1.0/server/player.c
--- crossfire-1.1.0.orig/server/player.c	Wed Mar 20 07:36:53 2002
+++ crossfire-1.1.0/server/player.c	Fri Mar 22 13:23:04 2002
@@ -57,7 +57,7 @@
   FILE *fp;
   int comp;
 
-  sprintf(buf,"%s/%s",settings.datadir,MOTD);
+  sprintf(buf,"%s/%s","/etc/crossfire",MOTD);
   if((fp=open_and_uncompress(buf,0,&comp))==NULL) {
     return;
   }
